CoffeeChat/Utilities/Time+Helper.swift

```diff
@@ -35,16 +59,119 @@ class Time {
     If today is October 4th 1:00 AM and the meeting is October 5th 11:00 PM this returns true
     */
     static func isTommorow(_ date: Date) -> Bool {
+        let calendar = Calendar.current
         if
-            let today = Calendar.current.date(bySettingHour: 0, minute: 0, second: 0, of: Date()),
-            let meetingDate = Calendar.current.date(bySettingHour: 0, minute: 0, second: 0, of: date)
+            let today = calendar.date(bySettingHour: 0, minute: 0, second: 0, of: Date()),
+            let meetingDate = calendar.date(bySettingHour: 0, minute: 0, second: 0, of: date)
         {
```

Comment:
This is inconsistent formatting. I think it should be

    if let today = ,
      let meetingDate =  {

    }

Reply:
(if you did this due to a swiftlint thing, we can update the .swiftlint.yml to not obey whatever rule it's complaining about)